### PR TITLE
Disable ProcessWatchdog

### PR DESCRIPTION
**Pull request by lgolding**
@jaredpar @jasonmalinowski @tmat @dotnet/roslyn-infrastructure @brettfo @amcasey

### CODE REVIEW

File: netci.groovy [hunk 1]
```diff
@@ -232,7 +232,7 @@ def branchNames = []
                     batchFile("""set TEMP=%WORKSPACE%\\Binaries\\Temp
 mkdir %TEMP%
 set TMP=%TEMP%
-.\\cibuild.cmd ${(configuration == 'dbg') ? '/debug' : '/release'} ${(buildTarget == 'unit32') ? '/test32' : '/test64'} /buildTimeLimit ${BuildTimeLimit}""")
```
Comment (lgolding) [inline netci.groovy]: The presence of the `/buildTimeLimit` switch on the cibuild.cmd command line is what enables the ProcessWatchdog.

**Comment by amcasey** [comment]
There's a bug somewhere tracking re-enabling this?

**Comment by amcasey** [comment]
LGTM

**Comment by lgolding** [comment]
@amcasey Not a bug _per se_. ProcessWatchdog was my task from "Infrastructure Week", there's still a sticky for it on our Kanban board, so I'll keep at it until it's done.

**Comment by jaredpar** [comment]
Why did we need to disable this?

**Comment by lgolding** [comment]
Nothing bad actually happened; just a precaution. Too long to go into here. We'll chat offline.

**Pull request merged**